rotkehlchen/chain/evm/tokens.py

```diff
@@ -175,6 +176,11 @@ def get_token_balances(
                     f'token balance for {address} and balance {normalized_balance}',
                 )
                 balances[Asset(token.identifier)] += normalized_balance
+        except RemoteError as e:
+            log.error(
+                f'{self.evm_inquirer.chain_name} tokensBalance call failed for address {address}.'
+                f' {tokens=}. Error: {e}',
+            )
```

Comment:
> but the error that casper saw was not RemoteError

yeah, the error that occurs is an `InvalidAddress` exception but that's not what's bubbled up. It's a `RemoteError` that gets bubbled up since it tries every possible node before bailing out.

> In that case you don't want to retry with other nodes

okey okey.